File: C/ATIVIDADE-5/ATIVIDADE-2.c
/*02. Faça um programa em C que leia o nome o endereço e o telefone de um cliente e ao final, imprima esses dados.
*/

#include <stdio.h>
#include <locale.h>

int main()
{
    setlocale(LC_ALL,"Portuguese");
   char nome[20];
   char endereco[50];
   char telefone[20];

   printf("Digite o seu nome: ");
   gets(nome);

   printf("Digite o seu endereco: ");
   gets(endereco);

   printf("Digite o seu telefone: ");
   gets(telefone);

   printf("%s\n",nome);
   printf("%s\n",endereco);
   printf("%s\n",telefone);
}

File: JAVA/ATIVIDADE-2/atividade2.java
import java.util.Scanner;

/*02. Faça um programa em Java que leia o nome o endereço e o telefone de um cliente e ao final, imprima esses dados.
*/
public class atividade2 {
    public static void main(String[] args) {
        String nome, endereco, telefone;

        Scanner entrada = new Scanner(System.in);

        System.out.print("Digite o seu nome: ");
        nome = entrada.nextLine();

        System.out.print("Digite o seu endereço: ");
        endereco = entrada.nextLine();

        System.out.print("Digite o seu telefone: ");
        telefone = entrada.nextLine();

        System.out.printf("NOME: %s \nENDEREÇO: %s \n TELEFONE: %s",nome,endereco,telefone);


        entrada.close();
    }
}
